mymodel/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<model>
+  <name>Wedge juggler</name>
```

Comment:
This looks like a copy from [this tutorial](http://gazebosim.org/tutorials?tut=model_contrib). Did you mean to modify it?